.infrastructure/helm-chart/todoapp/charts/mysql/templates/service.yml

```diff
@@ -0,0 +1,17 @@
+# Headless service for stable DNS entries of StatefulSet members.
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Chart.Name }}-headless-service
+  namespace: {{ .Values.common.namespace }}
+spec:
+  selector:
+    app: mysql
+  ports:
+  - name: mysql
+    port: 3306
+  clusterIP: None
+
+# pod-name.service-name.namespace.svc.cluster.local
+# pod-name.service-name
+# mysql-0.mysql.mysql.svc.cluster.local
```

Comment:
why is this comment needed? Also add EoF
![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_12_helm_charts/assets/87720583/06e6dedc-5f75-46c7-81df-9ae7b4f60eba)

Reply:
fixed